### PR TITLE
cat, cksum, comm, cut, chcon, csplit: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/cut.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Snij velden eruit vanuit `stdin` of bestanden.
+> Meer informatie: <https://www.gnu.org/software/coreutils/cut>.
+
+- Toon een specifiek karakter/veldbereik voor iedere regel:
+
+`{{commando}} | cut --{{characters|fields}}={{1|1,10|1-10|1-|-10}}`
+
+- Toon een bereik voor iedere regel met een specifieke scheiding:
+
+`{{commando}} | cut --delimiter="{{,}}" --{{fields}}={{1}}`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.nl/common/cut.md:12]: Can confirm this is a typo, feel free to update it in the English page too.

**Review by vitorhcl — APPROVED** [review]

File: pages.nl/osx/cut.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Snij velden eruit vanuit `stdin` of bestanden.
+> Meer informatie: <https://manned.org/man/freebsd-13.0/cut.1>.
+
+- Toon een specifiek karakter/veldbereik voor iedere regel:
+
+`{{commando}} | cut -{{c|f}} {{1|1,10|1-10|1-|-10}}`
+
+- Toon een bereik voor iedere regel met een specifieke scheiding:
+
+`{{commando}} | cut -d "{{,}}" -{{c}} {{1}}`
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages.nl/osx/cut.md]: In GNU cut, you cannot specify `-d` when also specifying `-d` -- I suppose this is different for the osx version?

**Review by kbdharun — APPROVED** [review]

**Review by vitorhcl — APPROVED** [review]

File: pages.es/osx/cut.md
Comment (kant) [inline pages.es/osx/cut.md:1]: LGTM

**Review by kant — APPROVED** [review]
Spanish translation: LGTM

File: pages.nl/osx/cut.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages.nl/osx/cut.md]: fixed in #12015

**Review by kbdharun — APPROVED** [review]

File: pages.nl/common/cksum.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# cksum
+
+> Berekn de CRC checksums en het aantal bytes van een bestand.
```
Comment (leonvsc) [inline pages.nl/common/cksum.md:3]:
```suggestion
> Bereken de CRC checksums en het aantal bytes van een bestand.
```

**Review by leonvsc — APPROVED** [review]
The Dutch part looks good to me after the suggestion I left!

**Pull request merged**